rush.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
   "rushVersion": "5.83.1",
   "pnpmVersion": "7.13.0",
-  "nodeSupportedVersionRange": ">=14.15.0 <15.0.0 || >=16.13.0 <17.0.0 || >=18.14.0 <19.0.0",
+  "nodeSupportedVersionRange": ">=16.13.0 <17.0.0 || >=18.14.0 <19.0.0",
```

Comment:
Since all our `@types/node` and lambda runtimes are Node 18, should this just be Node 18 as well?